validator/client/validator.go

```diff
@@ -939,8 +940,14 @@ func (v *validator) logDuties(slot types.Slot, duties []*ethpb.DutiesResponse_Du
 
 // UpdateFeeRecipient calls the prepareBeaconProposer RPC to set the fee recipient.
 func (v *validator) UpdateFeeRecipient(ctx context.Context, km keymanager.IKeymanager) error {
+	// only used after Bellatrix
 	if v.feeRecipientConfig == nil {
-		log.Warnln("Fee recipient config not set, skipping fee recipient update. Validator will continue proposing using beacon node specified fee recipient.")
+		e := params.BeaconConfig().BellatrixForkEpoch
+		if e != math.MaxUint64 && slots.ToEpoch(slots.CurrentSlot(v.genesisTime)) < e {
+			log.Warnln("Please plan for using the fee recipient flags post Bellatrix Hard Fork to receive transaction fee rewards on validator work!!")
+		} else {
+			log.Warnln("Fee recipient config not set, skipping fee recipient update!! Validator will continue proposing using beacon node specified fee recipient.")
```

Comment:
In order to receive transaction fees from proposing blocks, you must now specify a configuration known as a fee recipient config. If it not provided, transaction fees will be burnt. Please see our documentation for more information on this requirement <LINK>